pkg/csi/service/vanilla/controller.go

```diff
@@ -555,6 +587,18 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		}
 		resp.Volume.AccessibleTopology = append(resp.Volume.AccessibleTopology, volumeTopology)
 	}
+
+	// Set the Snapshot VolumeContentSource in the CreateVolumeResponse
+	if contentSourceSnapshotID != "" {
+		resp.Volume.ContentSource = &csi.VolumeContentSource{
+			Type: &csi.VolumeContentSource_Snapshot{
+				Snapshot: &csi.VolumeContentSource_SnapshotSource{
+					SnapshotId: contentSourceSnapshotID,
```

Comment:
Can you split this up, easier to read.

Reply:
can you elaborate? I am not sure what to split up.

Reply:
I meant define Type, Snapshot on different lines. Not very particular, I leave it up to you.